### ISSUE TITLE
Add phone editor + two factor authentication switch to default theme.

### ISSUE THREAD

**Issue by yecli** [VirtoCommerce/vc-theme-default#69]
###  User cabinet
**User stories**
- As **User** I want to  enable/disable Two Factor authentication in my cabinet
- As **User** I want to add / change my phone number only after confirming the code received via SMS

**Technical details:**

- Extend the current user cabinet "Profile" section in the default theme with two new fields
- Phone number (add, change, remove) and Two Factor Authentication   (on/off)
![image](https://user-images.githubusercontent.com/7566324/51819734-21661b00-22dc-11e9-9839-dae550614aa1.png)
![image](https://user-images.githubusercontent.com/7566324/51819637-caf8dc80-22db-11e9-93a1-fdef7ddbaebb.png)

API to use: 
POST storefrontapi/account/phonenumber - for add/change phone number (only after verification)
POST storefrontapi/account/phonenumber/verify - for verify phone number
DELETE storefrontapi/account/phonenumber - for delete phone number
POST storefrontapi/account/twofactorauthentification - for on/off 2FA

**Mockups**
Profile tab with no phone (modify assets\js\account\account-profile-update.tpl.liquid, fields PhoneNumber and TwoFactorEnabled):
![image](https://user-images.githubusercontent.com/15610352/51965426-55337300-248b-11e9-8edf-fbdd3fde8155.png)

Two factor switch disabled.

Adding phone number (add assets\js\account\phone-number.tpl.liquid):
![image](https://user-images.githubusercontent.com/15610352/51965173-7b0c4800-248a-11e9-80ff-dd9951e5a1ea.png)

Verifying number (add assets\js\account\verify-phone-number.tpl.liquid):
![image](https://user-images.githubusercontent.com/15610352/51965205-8d868180-248a-11e9-9f32-cc91ae126678.png)

When we have phone:
![image](https://user-images.githubusercontent.com/15610352/51965231-a2631500-248a-11e9-89e2-9b5abb5d923a.png)
Two factor auth switcher could be enabled.

Please use common storefront styling for components, these are just mockups with the main idea.
Add/edit/remove phone number logic should be implemented here in scripts too, assuming we have required answers from API. You can check addresses tab as an example.

**Comment by yecli** [comment]
Update:
Please get [feature/197-2FA-login-workflow](https://github.com/VirtoCommerce/vc-storefront-core/compare/feature/197-2FA-login-workflow) branch and add forms according to added API:
- GET /account/phonenumber (string phoneNumber);  (Called on Add and Edit phone)
- POST /account/phonenumber (UpdatePhoneNumberModel formModel);
Form "customers/phone_number", fields: customer[phoneNumber];
- GET /account/phonenumber/verify (string phoneNumber);  (Form to enter verification code)
- POST /account/phonenumber/verify (VerifyPhoneNumberModel formModel);
Form "customers/verify_phone", fields: customer[code], customer[phoneNumber];
- DELETE /account/phonenumber; (Called on Delete phone)
- POST /account/twofactorauthentification (EnableTwoFactorAuthenticationModel model); (Called on changing Two factor authentication).

Previously added views in assets\js\account\ are not needed now.

**Comment by yecli** [comment]
All changes were done in [#70 Support Two factor authentication in Login form PR](https://github.com/VirtoCommerce/vc-theme-default/pull/71)

**Comment by tatarincev** [comment]
Released https://github.com/VirtoCommerce/vc-theme-default/releases/tag/v2.0.0
